sunflower-land/sunflower-land
# [FIX] Easter event decorations dimensions

# Description

fixes for current easter event decorations:
- make sure collectibles
  - have minimized file sizes
  - have correct pixel size and pixel placement
  - not visually obstructing other island items that are outside their respective grids as much as possible
  - not obstructing click events that are outside their respective grids
  - has React.memo catch
- add/fix collectable dimensions

Before|After
---|---
![image](https://user-images.githubusercontent.com/107602352/231234582-a844cc4f-5c6d-4a47-a4f8-94d3e37036c6.png)|![image](https://user-images.githubusercontent.com/107602352/231234619-3b36d5a4-4810-4162-8ac7-1fa152fd892a.png)

## Type of change


# How Has This Been Tested?

- place easter event decorations (giant carrot, pablo the bunny, easter bush, easter bear)
- select rusty shovel and hover over these collectables and make sure they are only selectable in their respective squares

# Checklist:
